toolkit/app-write-mram.py

```diff
@@ -139,7 +139,9 @@ def main():
     parser = argparse.ArgumentParser(
         description="NVM Burner for Application TOC Package"
     )
-    parser.add_argument("-d", "--device", type=str, help="serial port device")
+    parser.add_argument(
+        "-p", "--port", type=str, help="Serial port device", required=True
```

Comment:
This won't work, it conflicts with the `-p` pad argument.

Because this option is required (options are usually *optional*!), maybe just make it a positional required argument?  Same with `maintenance.py` and `updateSystemPackage.py`.

Reply:
You're right I missed the `-p` for padding. We could just use `--port` or `-P`? I prefer explicit keyword args to positional args to match the other args style.

Reply:
But if you always have to specify it, why not make it positional and then you don't have to type `--port` each time?  It's pretty clean to write `maintenance.py /dev/ttyUSB0`.

> I prefer explicit keyword args to positional args to match the other args style.

The other arguments are all optional.  IMO it's more inconsistent to add a *required* option, rather than adding a positional argument.

Reply:
> But if you always have to specify it

You don't, it should have a default value like `/dev/ttyACM0`.

Reply:
OK, it's just that you added `required=True` here :joy:

Reply:
If it's positional, it doesn't need a unique name.

If it's an argument, we could skip the shorthand and call it `--device` or `--port`.

Reply:
Could use `-P` and `--port`.  Or remove the shorthand `-p` from the pad option and reuse `-p` for port.

Reply:
Okay, updated. I wasn't sure if we should keep `-p` for tools that don't have it, so I just removed it from all tools.

Reply:
> Could use -P and --port

No problem, I could revert that if you like.

Reply:
I think it's fine with just `--port`.  This tool will anyway most likely be used from a makefile or script.

Reply:
Let's get this merged and then focus on the db issue.